tests/facility/invoice/invoiceCreation.spec.ts

```diff
@@ -0,0 +1,163 @@
+      await expect(
+        page
+          .locator("li[data-sonner-toast]")
+          .getByText(/invoice created successfully/i),
+      ).toBeVisible({ timeout: 10000 });
+    });
+    await test.step("Verify status is draft and 'issue invoice' button is disabled", async () => {
+      await expect(page.getByText(/invoice: draft/i)).toBeVisible();
+      await expect(
+        page.getByRole("button", { name: /issue invoice/i }),
+      ).toBeDisabled();
+    });
+  });
+  test("should add charge items and issue invoice", async ({ page }) => {
+    await page.getByRole("link", { name: /see invoice/i }).click();
+    await test.step("Add charge items to invoice", async () => {
+      await expect(
+        page.getByRole("button", { name: /add charge item/i }),
+      ).toBeVisible();
+      await page.keyboard.press("a");
+      await expect(
+        page.getByRole("button", { name: /other charge items/i }),
+      ).toBeVisible();
+      await page.keyboard.press("o");
+      await page.getByRole("combobox").click();
+      await page.getByRole("option", { name: /tests/i }).first().click();
+      await page.getByRole("option", { name: /test/i }).first().click();
+      await expect(
+        page.getByRole("button", { name: /add items/i }),
+      ).toBeEnabled();
+      await page.keyboard.press("Enter");
+      await expect(
+        page
+          .locator("li[data-sonner-toast]")
+          .getByText(/charge items added successfully/i),
+      ).toBeVisible({
+        timeout: 10000,
+      });
+      await expect(
+        await page.getByRole("button", { name: /add selected items/i }),
+      ).toBeEnabled();
+      await page.keyboard.press("Shift+Enter");
+    });
+    await test.step("Issue invoice", async () => {
+      await expect(
+        page.getByRole("button", { name: /issue invoice/i }),
+      ).toBeEnabled();
+      await page.keyboard.press("i");
+      await expect(
+        page
+          .locator("li[data-sonner-toast]")
+          .getByText(/invoice updated successfully/i),
+      ).toBeVisible({ timeout: 10000 });
+      await expect(page.getByText(/invoice: issued/i)).toBeVisible();
+    });
+  });
+
+  test.afterAll(async ({ browser }) => {
+    const context = await browser.newContext({
+      storageState: "tests/.auth/user.json",
+    });
+    const page = await context.newPage();
+    const facilityId = getFacilityId();
+
+    await page.goto(`/facility/${facilityId}/encounters/patients/all`);
+
+    await page
+      .getByRole("button", { name: /view encounter/i })
+      .first()
+      .click();
+
+    await page
+      .locator("[data-slot='patient-info-hover-card-trigger']")
+      .last()
+      .click();
+
+    await page.getByRole("link", { name: /view profile/i }).click();
+    await page.getByRole("tab", { name: /accounts/i }).click();
+
+    await page
+      .getByRole("button", { name: /go to account/i })
+      .first()
+      .click();
```

Comment:
@GSMPRANEETH After extracting accountId, you can just go to an account like... instead of navigating from encounter -> patient -> account

```suggestion
    await page.goto(`/facility/${facilityId}/billing/account/${accountId}`);
```

Reply:
Implemented suggestion :+1: